tests/torch/quantization/test_functions.py

```diff
@@ -242,7 +207,7 @@ def to_tensor(a):
 
 
 def check_outputs_for_quantization_functions(test_val: torch.Tensor, ref_val: np.ndarray,
-                                             is_fp16: bool, rtol=1e-4, atol=1e-10):
```

Comment:
Was `is_fp16` unused before?

Reply:
No, even my IDE marked this as an unused parameter.